lib/locales/en.ts

```diff
@@ -789,6 +789,14 @@ export const locale: Locale = {
     our_lady_mother_of_christian_unity: 'Our Lady, Mother of Christian Unity',
     our_lady_mother_of_divine_providence_patroness_of_puerto_rico:
       'Our Lady, Mother of Divine Providence, Patroness of Puerto Rico',
+    /*
+     * src: https://gcatholic.org/calendar/2024/CR-en.htm
+     */
+    our_lady_of_angels: 'Our Lady of Angels',
```

Comment:
This should be removed, as it is not used in any calendar in romcal.

```suggestion
```

Reply:
That said, the `data:checks` script [passes](https://github.com/romcal/romcal/actions/runs/9568581543/job/26379148706?pr=546#step:9:12), which is strange to me.

Should it not raise a warning/error that `our_lady_of_angels` is not used? :thinking: Do the celebration IDs in calendars make these celebrations consider to be used? If so, we should update the `data:checks` script to:

- include all celebration IDs (the props of the `CalendarDef#inputs` object);
- exclude IDs of those celebration which have `customLocaleId` defined → then include the l10n ID which is set as the values of `customLocaleId`.

Reply:
no, it is used, it's just overridden in the one place it is technically used

Reply:
re: definite article in the name, yes. Both because the source uses "the angels" and the spanish literal uses `los Angeles` not `de Angeles`.

from the theological standpoint, doubly so. while we don't know why the Fallen angels fell, we know it had something to do with the Incarnation, to which the Theotokos is essential.

Reply:
Re definite article: I’ve read a bit about it. The actual (unofficial?) venerated title of Our Lady is _La Negrita_ (the Black One or Black Lady), however, the celebration name is actually coming from Porciuncula (now the Basilica of Our Lady of the Angels / Santa Maria degli Angeli) in Italy, via fransciscans.

I believe that Our lady is the Queen of _all_ (good/non-fallen) angels. I believe that while in English, we usually use the definite article which concreate stuff (among other things), we also use it when we talk about _all angels_. I could be wrong though. :wink:

Reply:
Hmmm, to me.the distinction when the noun is pluralized (of Angels/of the Angels) is moot, given its already applying to the plural. They would, in my reading, mean the same thing. A parallel is the Latin term `Panis Angelicus` denotes, to an English translation, Bread of Angels and Bread of the Angels equivalently.

Reply:
Indeed, the distiction is minimal and depends much on the context and the person who uses a particular phrase.

> A parallel is the Latin term `Panis Angelicus` denotes, to an English translation, Bread of Angels and Bread of the Angels equivalently.

While I get what you mean, I think there is no such _explicit_ distinction in Latin (as there is no in-/definite article in Latin), thus the parallel is in the _translation_ level only IMHO. That said, you are a native English speaker, not me. :wink: